org.eclipse.m2e.core/lifecycle-mapping-metadata.xml

```diff
@@ -199,6 +199,20 @@
         <ignore/>
       </action>
     </pluginExecution>
+    <pluginExecution>
+      <pluginExecutionFilter>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-toolchains-plugin</artifactId>
+        <versionRange>[2.0,)</versionRange>
```

Comment:
Do you have a specific reason for this version range?
Looking at M-C, there is no version 2 at all:
https://repo1.maven.org/maven2/org/apache/maven/plugins/maven-toolchains-plugin/

Reply:
No specific version, just grobe Kelle.